TM1py/Services/CellService.py

```diff
@@ -411,7 +411,15 @@ def write(self, cube_name: str, cellset_as_dict: Dict, dimensions: Iterable[str]
     def write_through_unbound_process(self, cube_name: str, cellset_as_dict: Dict, increment: bool = False,
                                       sandbox_name: str = None, precision=8, **kwargs):
         if sandbox_name:
-            raise NotImplementedError("Function does not support writing to sandboxes yet")
+            enable_sandbox = f"IF(ServerSandboxExists('{sandbox_name}')=1); " \
```

Comment:
I am thinking that perhaps we could do this check through the `SandboxService` if a sandbox exists, while still in python.
Then we wouldn't need to do it in every TI and we could throw a descriptive ValueError or something in python.

I am thinking this because I am currently working on a small change to add 16K CellPut lines to the prolog and another 16K into the epilog.
Our changes could be merged more harmoniously if the sandbox checks were not happening in the prolog.

Reply:
Moved to #451